testProjects/unitTestProject/src/main/scala/scalaclean/test/references/allOutgoingReferences/allOutgoingReferences.scala.expected

```diff
@@ -34,11 +34,11 @@ object XX {
   println(Special.myVar2)
 
   Special()
-  val x: Any = 1/* allOutgoingReferences(G:scalaclean/test/references/allOutgoingReferences/XX.x.) - <root>/scala :: <root>/scala/Any# */
+  val x: Any = 1/* allOutgoingReferences(<root>/scalaclean/test/references/allOutgoingReferences/XX/x .) - <root>/scala :: <root>/scala/Any# */
```

Comment:
I think the trailing space is from nsc, where it's some kind of indicator of "local name", I think?

Reply:
Yes - its a filed. We should probably handle that specially, like we do for methods

Reply:
filed?

Reply:
field